src/EnergyPlus/ThermalComfort.cc

```diff
@@ -715,6 +718,16 @@ namespace ThermalComfort {
 			} else {
 				ShowSevereError( "PEOPLE=\"" + People( PeopleNum ).Name + "\", Incorrect Clothing Type" );
 			}}
+			// for unit testing
```

Comment:
I get that it doesn't affect any other code, but I share this concern with Mike.  I think it is more of a philosophical issue that a whether it works issue.  I think it is best to keep the actual E+ free from unit testing stuff and keep the unit testing code with the unit test.  However, I do not know what if any rules or precedents have been set so I think we need some sort of ruling from the code cop.  @Myoldmopar What rules have been established for this?